tests/test_transcription.py

```diff
@@ -0,0 +1,90 @@
+# CREATED: 2/9/16 2:27 PM by Justin Salamon <justin.salamon@nyu.edu>
+
+import mir_eval
+import numpy as np
+
+A_TOL = 1e-12
+
+
+def _load_unit_test_reference():
```

Comment:
Ok, getting extremely pedantic here (sorry!) but I don't see any reason to have these as separate "load" functions now.